cmd/security-agent/app/runtime.go

```diff
@@ -129,5 +130,10 @@ func startRuntimeSecurity(hostname string, endpoints *config.Endpoints, context
 
 	log.Info("Datadog runtime security agent is now running")
 
+	// Send the runtime 'running' metrics periodically
+	ticker := time.NewTicker(15 * time.Second)
```

Comment:
Should we maybe define the ticker in the `sendRunningMetrics` function so that it is defined only once ?

Reply:
I changed the way the Ticker is initialized. Could you have another look?